state/applicationoffers.go

```diff
@@ -243,13 +243,23 @@ func (op *RemoveOfferOperation) internalRemove(offer *crossmodel.ApplicationOffe
 		Id:     offer.ApplicationName,
 		Assert: bson.D{{"relationcount", app.doc.RelationCount}},
 	}}
+	var remoteApps []*RemoteApplication
```

Comment:
I don't see why we need this slice of remote apps?
We add to it but it's not used.

Reply:
Good catch. This was an artifact of a previous approach that I didn't cull completely. Removed.